<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="marufpc2" name="Test101" tests="1" failures="1" timestamp="14 May 2020 23:55:10 GMT" time="0.056" errors="0">
  <testcase name="q" time="0.026" classname="test1.a">
    <failure type="java.lang.AssertionError" message="expected [5] but found [4]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [4]
	at test1.a.q(a.java:10)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- q -->
</testsuite> <!-- Test101 -->
